qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/customers/CustomerDbConnector.groovy

```diff
@@ -75,7 +83,6 @@ class CustomerDbConnector implements CustomerDbGateway {
   @Override
   void updateCustomer(String customerId, Customer updatedCustomer) {
     databaseQueries.updateCustomer(customerId,updatedCustomer)
```

Comment:
can there also occur an exception that needs to be thrown?

Reply:
not part of this Use Case

Reply:
Fair enough